elevation_mapping/src/sensor_processors/SensorProcessorBase.cpp

```diff
@@ -158,5 +163,37 @@ void SensorProcessorBase::removePointsOutsideLimits(
   ROS_DEBUG("removePointsOutsideLimits() reduced point cloud to %i points.", (int) pointClouds[0]->size());
 }
 
+bool SensorProcessorBase::cleanPointCloud(const pcl::PointCloud<pcl::PointXYZRGB>::Ptr pointCloud)
+{
+  pcl::PassThrough<pcl::PointXYZRGB> passThroughFilter;
+  pcl::PointCloud<pcl::PointXYZRGB> tempPointCloud;
+
+  // remove nan points
```

Comment:
Please use proper formatting for comments.

Reply:
What do you mean by proper formatting?